tests/Filter/DateTimeFilterTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Filter;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery;
+use Sonata\DoctrineORMAdminBundle\Filter\DateTimeFilter;
+use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
+
+/**
+ * @author Marko Kunic <kunicmarko20@gmail.com>
+ */
+class DateTimeFilterTest extends TestCase
+{
+    public function testEmpty()
+    {
+        $filter = new DateTimeFilter();
+        $filter->initialize('field_name', ['field_options' => ['class' => 'FooBar']]);
+
+        $builder = new ProxyQuery(new QueryBuilder());
```

Comment:
This queryBuilder should have a `use` right?

Reply:
There is a QueryBuilder class i that folder https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/tests/Filter/QueryBuilder.php